ocm-container.sh

```diff
@@ -4,11 +4,12 @@ usage() {
   cat <<EOF
   usage: $0 [ OPTIONS ] [ Initial Cluster Name ]
   Options
-  -e  --exec          Path (in-container) to a script to run on-cluster and exit
-  -h  --help          Show this message and exit
-  -o  --launch-opts   Sets extra non-default container launch options
-  -t  --tag           Sets the image tag to use
-  -x  --debug         Set bash debug flag
+  -e  --exec          		Path (in-container) to a script to run on-cluster and exit
+  -h  --help          		Show this message and exit
+  -o  --launch-opts   		Sets extra non-default container launch options
+  -t  --tag           		Sets the image tag to use
+  -x  --debug         		Set bash debug flag
+  -d --disable-console-port   	Disable automatic cluster console port mapping
```

Comment:
Can we add a bit of text here to say that this has no effect on MacOS at this time, or that the console feature will not work on MacOS/docker?